tools/wheels/cibw_before_build_linux.sh

```diff
@@ -17,6 +17,18 @@ printenv
 # Update license
 cat $PROJECT_DIR/tools/wheels/LICENSE_linux.txt >> $PROJECT_DIR/LICENSE.txt
 
+# TODO: delete along with enabling build isolation by unsetting
+# CIBW_BUILD_FRONTEND when scipy is buildable under free-threaded
+# python with a released version of cython
+FREE_THREADED_BUILD="$(python -c"import sysconfig; print(bool(sysconfig.get_config_var('Py_GIL_DISABLED')))")"
+if [[ $FREE_THREADED_BUILD == "True" ]]; then
+    python -m pip install -U --pre pip
+    python -m pip install git+https://github.com/cython/cython
+    python -m pip install -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple numpy
+    python -m pip install git+https://github.com/serge-sans-paille/pythran
```

Comment:
Are there any known issues in `pythran`? If not, easier to add `pythran` to the next line next to `pybind11`.